db/migrate/20161209113503_add_timestamps_to_bookable_slots.rb

```diff
@@ -0,0 +1,15 @@
+class AddTimestampsToBookableSlots < ActiveRecord::Migration[5.0]
+  class BookableSlot < ApplicationRecord; end
+
+  def change
+    add_timestamps :bookable_slots, null: true
+
+    BookableSlot.update_all(
+      created_at: Time.zone.now,
+      updated_at: Time.zone.now
+    )
```

Comment:
Should we store a reference to `Time.zone.now` up above, so that `created_at` and `updated_at` are exactly the same?